internal/commands/create.go

```diff
@@ -199,28 +185,31 @@ func getConstraint(policy rego.File) (unstructured.Unstructured, error) {
 		}
 	}
 
-	policyCommentBlocks, err := getPolicyCommentBlocks(policy.Comments)
-	if err != nil {
-		return unstructured.Unstructured{}, fmt.Errorf("get policy comment blocks: %w", err)
-	}
-
-	if len(policyCommentBlocks) == 0 {
+	matchers := GetMatchersFromComments(policy.Comments)
+	if len(matchers.KindMatchers) == 0 {
 		return constraint, nil
 	}
 
 	var kinds []interface{}
 	var apiGroups []interface{}
-	for _, policyCommentBlock := range policyCommentBlocks {
-		for _, policyKind := range policyCommentBlock.Kinds {
-			kinds = append(kinds, policyKind)
+	for _, kindMatcher := range matchers.KindMatchers {
+		kinds = append(kinds, kindMatcher.Kind)
+	}
+
+	for _, kindMatcher := range matchers.KindMatchers {
+		apiGroup := kindMatcher.APIGroup
+		if kindMatcher.APIGroup == "core" {
+			apiGroup = ""
 		}
 
-		for _, policyAPIGroup := range policyCommentBlock.APIGroups {
-			if policyAPIGroup == "core" {
-				policyAPIGroup = ""
+		var exists bool
```

Comment:
This is something I don't think I'll be able to bring myself to do :rofl:

In my opinion it's more "Go-y" to let Go set the zero value for you, rather than you setting it yourself.
If we don't know exactly what the value should be, then we should declare it and use its zero value 

```golang
var anything string
```

If we know exactly what the value should be, then we set it via `:=`

```golang
initialValue := "The page was not found"
```

Reply:
I disagree. I think it's better to be clear, especially when the compiler should generate the exact same instructions for the two ways of writing it. However this is a nit, so really either way is fine.